boba/lang.py

```diff
@@ -4,57 +4,65 @@
 PY = 'python'
 R = 'r'
 
+script = '{{script_name}}'
+compiled = '{{universe_name}}'
+
+DEFAULT_LANGS = {
+    'python' : {
+        'ext' : ['py'],
+        'run' : ['python', script]
+    },
+    'r' : {
+        'ext' : ['R', 'r', 'rmd'],
```

Comment:
we can't really use `Rscript` to run rmd if it contains custom syntax like \`\`\`{r}```